src/lib/navbar.svelte

```diff
@@ -27,7 +30,34 @@
 			>
 				<Logo type="dark" />
 			</div>
-			<div class="flex items-center">
+			<div class="md:hidden">
+				<Menu>
+					<div class="flex flex-col">
+						<MenuItem class="hover:bg-primary-100">
+							<IconButton on:click={()=>{window.open("https://blog.openregistry.dev",'_blank')}}> 
+							 Blog
+							</IconButton>
+						</MenuItem>
+						<MenuItem class="hover:bg-primary-100">
+							<IconButton on:click={()=>{window.open("/about",'_self')}}> 
+							 About
+							 </IconButton>
+						</MenuItem>
+						<MenuItem class="hover:bg-primary-100">
+							<IconButton on:click={()=>{window.open("https://github.com/containerish/OpenRegistry.git",'_blank')}}> 
+								Github
+							</IconButton>
+						</MenuItem>
+						<MenuItem class="hover:bg-primary-100">
```

Comment:
@guacamole let's also add a signin button here? let's only do signin and not signup